Cargo.toml

```
@@ -11,25 +11,25 @@ nightly = []
 all = ["rocksdb"]
 
 [dependencies]
-anyhow = "1.0.26"
+anyhow = "1.0.28"
 async-std = { version = "1.5.0", features = ["attributes", "std"] }
-async-trait = "0.1.24"
+async-trait = "0.1.29"
 bitswap = { path = "bitswap" }
 byteorder = "1.3.4"
 dirs = "2.0.2"
 domain = { git = "https://github.com/nlnetlabs/domain", features = ["resolv"] }
 futures = { version = "0.3.4", features = ["compat", "io-compat"] }
-libipld = { version = "0.1.0", features = ["dag-pb"] }
+libipld = { version = "0.1.0", features = ["dag-json", "dag-pb"], path = "../rust-ipld" }
```

```suggestion
libipld = { git = "https://github.com/ipfs-rust/rust-ipld", features = ["dag-pb"] }
```
Took me a while to understand the CI failure, but I happened to have a recent checkout in the same path :)

---

Aha!

---

Forgive me @dvc94ch but I am going to commit this so I can run conformance testing on this branch

---

@aphelionz there's also a way to patch a dependency https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-patch-section which might be a clearer indicator of doing something temporarily.